### DISCUSSION TITLE
Images way too bright now - washed out

### DISCUSSION THREAD

**Discussion by jmar6482** [ImageMagick/ImageMagick#7497] [Help] [answered]
### ImageMagick version

7.1

### Operating system, version and so on

Linux

### Description

am having problems running imagemagick and causing images that are way too bright.
I use imagemagick on an old Website script called ImageFolio, that is no longer supported, to batch create Thumbnails and Preview Images sized to my specs. The process involves placing the High Resolution images in a particular folder. I invoke the script pointing to that folder and Imagemagick does its thing thereby populating the site with resized images (thumbs, previews and renamed original size files. However, although the images are re-sized, renamed correctly etc., the thumbnail and previews created are now way too light and unusable. I don't make any specific instructions to change the way the images look, I only create thumbs and previews with no change in appearance. Are there any known issues in using the latest version of Imagemagick as opposed to older versions that  I was using before, on my old windows server/host. version 6.6.9? It no longer works for me on Hostgator that's hosting a similar site

Now that I migrated to Dreamhost using linux and the latest version of imagemagick I get these unusable, very light, images.

Any idea how to make this work properly? 

This script worked very well for many many years both on my home windows server and at Hostgator. Then up until about March of 2021 Hostgator upgraded  ImageMagick on its servers and I started getting the extremely bright images.
I am now abandoning my home server and now must rely on Bluehost and Hostgator to host the sites that are using this script. Consequently I need this to work properly now more that ever. Any help would be greatly appreciated. I am not really that technical so help in simple terms would be appreciated.

**Comment by fmw42** [comment]
We will have to see the script or at least the command line(s) where you do your Imagemagick processing

Also your exact version of Imagemagick 7.1.x.x and what you get from 

`magick -version`

**Reply by jmar6482** [reply]
[imageBatch.pl.txt](https://github.com/user-attachments/files/16317484/imageBatch.pl.txt)
[imagemagick.pl.txt](https://github.com/user-attachments/files/16317485/imagemagick.pl.txt)
Hope the attached files help. I added the txt extension so they could be uploaded, they are perl files
the shared server has the following version installed:  Version: ImageMagick 6.9.10-23 Q16 x86_64 20190101

**Reply by fmw42** [reply]
Sorry, I do not know Perl. But I am suspicious of the line in youImagemagick.pl script:

`   $i = $image->Set(colorspace=>'RGB');                                         &Error("Error Setting Colorspace: $i") if $i;
`

<br>

At some point in IM 6 releases, colorspace RGB was swapped with colorspace sRGB.  The more current versions of IM would want that line to be

`   $i = $image->Set(colorspace=>'sRGB');                                         &Error("Error Setting Colorspace: $i") if $i;
`

<br>

Give that a try and let us know.

**Reply by jmar6482** [reply, marked as answer]
Thank YOU! That did it...
